.github/workflows/test.yaml

```
@@ -53,7 +53,7 @@ jobs:
         strategy:
             matrix:
                 os: [ubuntu-latest, macos-latest]   # eventually add `windows-latest`
-                python-version: [3.8, 3.9, "3.10"]
+                python-version: "3.10"
```

we support 3.9-3.12 now
```suggestion
                python-version: [3.9, "3.10", "3.11", "3.12"]
```